src/components/pages/PageProductForm/PageProductForm.tsx

```diff
@@ -109,7 +109,7 @@ export default function PageProductForm() {
   const onSubmit = (values: FormikValues) => {
     const formattedValues = ProductSchema.cast(values);
     const productToSave = id ? {...ProductSchema.cast(formattedValues), id} : formattedValues;
-    axios.put(`${API_PATHS.bff}/product`, productToSave)
+    axios.post(`${API_PATHS.bff}/products`, productToSave)
```

Comment:
UI sends POST request to https://4up6kagkma.execute-api.eu-west-1.amazonaws.com/dev/product
Maybe redeployment is needed? Please fix to gain +1 :)
<img width="1456" alt="Screenshot 2020-11-09 at 20 44 35" src="https://user-images.githubusercontent.com/7134713/98588737-77972900-22cc-11eb-8ff7-791fad572d99.png">

Reply:
I updated my pr. Please review that.

Reply:
Works good, +1